docs/detections/api/lists/lists-api-overview.asciidoc

```diff
@@ -54,19 +54,12 @@ Use an <<exceptions-api-create-exception-item, exception item>> to define the
 operator and associate it with an <<exceptions-api-create-container, exception container>>.
 You can then add the exception container to a rule's `exceptions_list` object.
 
-IMPORTANT: Before you can create lists, you must create `.lists` and `.items`
-data streams for the {kib} space (see <<lists-index-api-overview>>).
-
 [float]
-=== Kibana role requirements
+=== Lists prerequisites
 
-To create list containers and items, the user role for the {kib} space must
-have:
+Before you can start using lists, you must create the `.lists` and `.items` data streams for the relevant {kib} space. For more information, refer to <<lists-index-api-overview>>. 
 
-* `read` and `write` index privileges for the
-`.lists` and `.items` data streams (the system data stream used for storing exception lists).
-* {kib} space `All` privileges for the `Security` and `Saved Objects Management`
-features (see
-{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).
+[float]
+=== Required privileges 
 
-See <<detections-permissions-section>> for a complete list of requirements.
+To create list containers and list items, your role requires privileges to manage lists. Refer to <<enable-detections-ui>> for more information.  
```

Comment:
great to have it as reference, so all relevant info would be stored in one place 👍